core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/cli/TemplateToInstanceTemplateV4RequestConverter.java

```diff
@@ -44,7 +44,7 @@ private RootVolumeV4Request getRootVolume(Template source) {
 
     private VolumeV4Request getAttachedVolume(VolumeTemplate source) {
         VolumeV4Request ret = new VolumeV4Request();
-        ret.setCount(source.getVolumeCount());
+        ret.setCount(source.getVolumeCount() == null ? Integer.valueOf(0) : source.getVolumeCount());
```

Comment:
I think there is no need to box the value of 0 explicitly. You can simply write `0`.

Reply:
the setCount() function accepts Integer that's why I thought sending object is more appropriate.

Reply:
@SubhanHakverdiyev Java compiler does auto-boxing/-unboxing between primitive types and their corresponding wrapper classes (int - Integer in this case) in compile time, hence you can simply use 0

Reply:
Update: Apparently, it failed one of checks(namely spotbugs-main) as "_Boxed value is unboxed and then immediately reboxed_" . As source.getVolumeCount() is Integer it will be unboxed to compare it with primitive type 0 and later because ret.setCount() accepts Integer it will be boxed again so this caused performance warning. I have added Integer.valueOf again for that reason.